src/wasm-lib/kcl/src/std/mod.rs

```diff
@@ -235,6 +238,10 @@ impl Args {
         )?))
     }
 
+    fn make_user_val_from_point2d(&self, p: [f64; 2]) -> Result<MemoryItem, KclError> {
+        self.make_user_val_from_json(serde_json::json!([p[0], p[1]]))
```

Comment:
I'd just inline this in the stdlib function itself, only because its hard to know when these exist and makes the code more readable but just a nit no biggie

Reply:
You can also do `MemoryItem::UserVal {value: serde_json::to_value([p1, p2])?, meta:Default::default()}` does the same thing without the helper function

Reply:
might make it more clear

Reply:
also avoids the extra conversion

Reply:
Hell yeah thank you

Reply:
I couldn't quite use your suggestion because the compiler was telling me that I needed to explicitly map the serde error to a KCLError but I was able to find other examples that helped me do that

Reply:
just looked you def did the right thing :)